convert2rhel/subscription.py

```diff
@@ -973,8 +973,8 @@ def check_needed_repos_availability(repo_ids_needed):
         if repo_id not in avail_repos:
             # TODO: List the packages that would be left untouched
             loggerinst.warning(
-                "%s repository is not available - some packages"
-                " may not be replaced and thus not supported." % repo_id
+                "The '%s' repository is not available. Some packages may not be replaced with"
+                "their corresponding RHEL packages when converting." % repo_id
```

Comment:
This still (or again) has the 'withtheir' problem. Also, crazy long lines.

```suggestion
                "The '%s' repository is not available."
                " Some packages may not be replaced with their corresponding"
                " RHEL packages when converting." % repo_id
```

Reply:
Thank you, I will go back and change the formatting of this message!